### DISCUSSION TITLE
Support for migrating blogs

### DISCUSSION THREAD

**Discussion by haroldvandekamp** [WikiTransformationProject/wikitraccs-releases#56] [Ideas]
A customer is having their Confluence intranet almost 100% based on blogs. It looks like these aren't migrated yet. Is that correct?

Can support for migrating blogs be added?

**Comment by heinrich-ulbricht** [comment]
Technically Confluence blogs should migrate well to SharePoint, although demand so far was low. Let me check and get back to you, this should be clarified.

**Comment by haroldvandekamp** [comment]
That would be great.

Our customer has approximately 80% of their content based on blogs. Their news, internal vacancies and some other items types are all based on blogs. So, not being able to migrate them would be a huge loss for them.

Hope you can find a solution for migrating blogs. 🙂


________________________________
Van: Heinrich Ulbricht ***@***.***>
Verzonden: Tuesday, June 13, 2023 11:10:15 PM
Aan: WikiTransformationProject/wikitraccs-releases ***@***.***>
CC: Harold van de Kamp ***@***.***>; Author ***@***.***>
Onderwerp: Re: [WikiTransformationProject/wikitraccs-releases] Support for migrating blogs (Discussion #56)


Technically Confluence blogs should migrate well to SharePoint, although demand so far was low. Let me check and get back to you, this should be clarified.

**Comment by heinrich-ulbricht** [comment]
@haroldvandekamp You can give https://github.com/WikiTransformationProject/wikitraccs-releases/releases/tag/v1.5.0 a try - blog post migration can now be toggled on in the settings.

**Reply by haroldvandekamp** [reply]
That is great news @heinrich-ulbricht ! I'll try this out later this week.

**Reply by haroldvandekamp** [reply]
@heinrich-ulbricht First impression is good! 

Can you make sure that the blogs are published as news? Or provide a setting to make it possible to configure this? This allows us to differentiate between pages and news (blogs).

Technically, a page can have a promoted state of 0,1 and 2. Page = 0, News = 2 and can be viewed as a page property. Via the GUI this cn be changed via the Promote as News option:
![image](https://github.com/WikiTransformationProject/wikitraccs-releases/assets/11357196/1ad096e8-c9af-44f7-b20f-32e041b1b7b6)

**Reply by heinrich-ulbricht** [reply]
Should be doable. @haroldvandekamp Is setting the promoted state enough? Or does the parent content type need to change as well? Don't know from the top of my head, but I think promoted state should be enough.

I created https://github.com/WikiTransformationProject/wikitraccs-releases/issues/62 to track this.

**Reply by haroldvandekamp** [reply]
@heinrich-ulbricht I am pretty sure that only changing the promoted state is enough.

**Reply by heinrich-ulbricht** [reply]
@haroldvandekamp Promoting blog posts by default to SharePoint news posts has been added to the latest release.

**Reply by haroldvandekamp** [reply]
That is great news @heinrich-ulbricht! Thanks
